src/backend/agent.js

```diff
@@ -492,7 +494,9 @@ export default class Agent extends EventEmitter {
     window.removeEventListener('click', this._onClick, true);
     window.removeEventListener('mousedown', this._onMouseDown, true);
     window.removeEventListener('mouseup', this._onMouseUp, true);
-    window.removeEventListener('mouseover', this._onMouseOver, true);
```

Comment:
Should we still block "mouseover" events like we do for the other mouse/click events?

Reply:
Good point, added that back in, plus `mouseenter` and `pointerenter` to capture other cases.